ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -772,6 +1172,8 @@ KEXT_STATIC int HandleVnodeOperation(
 
     
 CleanupAndReturn:
+    eventTracer.SetVnodeOpResult(kauthResult);
```

Comment:
Would it be helpful to have the pathname here or is it easy enough to match up?

Reply:
The buffer in the `eventTracer` will already contain the path at this point, so it'll be printed together with the return value.